clarity_epp/export/magnis.py

```diff
@@ -15,11 +15,11 @@ def samplesheet(lims, process_id, output_file):
         well_strip[placement] = artifact.name
 
     # Write Header
-    output_file.write('sample_id\tinstrument_name\n')
+    output_file.write('sampleInstrument\n')
```

Comment:
Should it be Instrument or Instrumentname?

Reply:
Nop decide to remove the '_' and for clarity use 1 word per label (sample and instrument).